privatelink/aws/dns-endpoints.sh

```diff
@@ -13,20 +13,24 @@
 #     *.usw2-az1                CNAME vpce-0123456789abcdef0-01234567-us-west-2a.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
 #     *.usw2-az3                CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
 #
+#   % ./dns-endpoints.sh vpce-0123456789abcdef0 usw2-az3
+#     *                         CNAME vpce-0123456789abcdef0-01234567-us-west-2c.vpce-svc-0123456789abcdef0.us-west-2.vpce.amazonaws.com
 
 aws 1>/dev/null 2>/dev/null
 [[ $? == 127 ]] && echo "warning: please install 'aws'"
 
-if [[ $# != 1 ]]; then
-    echo "usage: $0 <VPC Endpoint>" 1>&2
+if [[ $# < 1 || $# > 2 ]]; then
+    echo "usage: $0 <VPC Endpoint> [<Availability Zone>]" 1>&2
```

Comment:
```suggestion
    echo "usage: $0 <VPC Endpoint> [<Availability Zone ID>]" 1>&2
```

Sorry, missed this in the first review.  We should be clear that we are asking for an Availability Zone ID.